.env

```diff
@@ -9,3 +9,5 @@ VITE_USER_SERVICE_BACKEND_URL=http://localhost:5001
 VITE_COLLAB_SERVICE_WS_BACKEND_URL=ws://localhost:5004
 VITE_COLLAB_SERVICE_BACKEND_URL=http://localhost:5004
 VITE_HISTORY_SERVICE_BACKEND_URL=http://localhost:5005
+VITE_HISTORY_SERVICE_CREATE_QUESTION_BACKEND_URL=http://localhost:5005/create-question-attempted
+VITE_HISTORY_SERVICE_STORE_USER_CODE_BACKEND_URL=http://localhost:5005/store-user-executed-code
```

Comment:
@ekweirui do we do it as such all do we pass it in the form of

`url, functionName,other params i dont rmb`

Reply:
yaa ok! have done so tq :)